util.go

```diff
@@ -10,8 +10,8 @@ const fieldTagName = "toml"
 
 // fieldCache maps normalized field names to their position in a struct.
 type fieldCache struct {
-	named map[string]fieldInfo // fields with an explicit name in tag
-	auto  map[string]fieldInfo // fields with auto-assigned normalized names
+	named map[string]*fieldInfo // fields with an explicit name in tag
```

Comment:
No need to use pointers. Tested PR without and it works great.

Reply:
i've added you to the timob/toml collaborators, can you push the changes to timob:master ?